scripts/FinalThresholdWorkout.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Threading.Tasks;
+
+class ThresholdWorkout {
+    // Define the constants for the workout
+    const int IntervalDuration = 30; // in seconds
+    const int NumIntervals = 2;
+    const int RestDuration = 20; // in seconds
+    const int WarmupDuration = 25; // in seconds
+    const int CooldownDuration = 15; // in seconds
+
+    // Define the power output for each part of the workout
+    static int ThresholdPower;
+    static int IntervalPower = 0; // initialize to 0
+    const int RestPower = 100; // in watts

```

Comment:
these are magic numbers, what if a users max FTP was only 50? this implementation would set the rest period to higher than their max load!

I would suggest setting these values to a specific percentage of the threshold power, rather than definitive values